.editorconfig

```diff
@@ -0,0 +1,19 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
this is 100% optional, it just makes my editor (and many other editors) format the files correctly.